.github/workflows/ceph-s3-tests.yaml

```diff
@@ -77,3 +75,21 @@ jobs:
             echo "The s3 tests did not run!"
             exit 1
           fi
+
+      - name: Collect logs
+        if: ${{ failure() }}         
+        run: |
+         set -x
+         echo "K8S Events"
+         kubectl get events --sort-by='.metadata.creationTimestamp' -A
+         mkdir -p my_logs
+         cd ./noobaa-operator
+         ./build/_output/bin/noobaa-operator diagnose --db-dump --dir=my_logs
+         cd ../my_logs
+         pwd
+         ls
+      - name: Save logs
+        uses: actions/upload-artifact@v3
```

Comment:
shouldn't  `if: ${{ failure() }}` be used here too?

Reply:
Should be, thanks